nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plot_series(df.tail(200), forecast_df, level=[80,95])
</pre><p>It's easier to use the max_insample_length argument here, e.g.</p><p><code>plot_series(df, forecast_df, max_insample_length=200, level=[80, 95])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>